moveit_core/CMakeLists.txt

```diff
@@ -1,14 +1,18 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(moveit_core VERSION 2.0.0 LANGUAGES CXX)
 
+unset(CATKIN_INSTALL_INTO_PREFIX_ROOT)
```

Comment:
Where's this from? Do you have a mixed ament/catkin setup?

Reply:
It also shows up in the [CI result](https://travis-ci.org/ros-planning/moveit2/builds/606100580#L2766). According to [here](https://github.com/colcon/colcon-ros/issues/83), seems it is used with catkin_tools upon colcon workspace. I searched through the code of moveit_core, haven't figured out how it is triggered on.